geode-core/src/test/java/org/apache/geode/management/internal/api/LocatorClusterManagementServiceTest.java

```diff
@@ -132,7 +132,8 @@ public void create_persistenceIsNull() {
 
   @Test
   public void create_validatorIsCalledCorrectly() {
-    doReturn(Collections.emptySet()).when(memberValidator).findServers(anyString());
+    doReturn(Collections.emptySet()).when(memberValidator);
+    memberValidator.findMembers(eq(false), new String[] {anyString()});
```

Comment:
This is very puzzling use of Mockito. Are these two lines trying to set up `findMembers()` to return an empty set? I haven't experimented with this, but I don't this does the right thing. Without a function call after the `when()`, I don't think the `doReturn()` does anything. I don't think the call to `findMembers()` also does anything. And I don't think the argument matchers (`eq()` and `anyString()`) are doing anything.

I think the net result of all of this is that you have not mocked `findMembers()` at all. So mockito, by default, returns an empty collection when the method is called.

And why this odd mocking syntax? Is it because of the varargs? Perhaps it would be better to create a non-varargs version or two—one that takes a single string, and another that takes an iterable or collection.

Reply:
those should be one line, I am not sure since when they become 2 lines of code.